### PR TITLE
Set up usefull style

### PR DESCRIPTION
**Pull request by twizerim**
### What does this pr do ?

This pr  is a configuration of css style used in our medica app , that type of css is nativewind.

### How to intall nativewind ?

       npm install nativewind
       npm install --save-dev tailwindcss@3.3.2

   or
     yarn add nativewind
    yarn add --dev tailwindcss@3.3.2

###   How is tailwind.config.js ?

      You must run npx tailwindcss init

### Modify the babel plugn
plugins: ["nativewind/babel"],

### nativewind-env.d.ts

/// <reference types="nativewind/types" /> to use templent typescripty

### CODE REVIEW

**Review by chrysloi — COMMENTED** [review]
- would you consider to setup the color palette we'll be using?
- also consider rebasing your PR!

**Comment by 911samuel** [comment]
consider first solving conflicts

**Comment by justinmihigo** [comment]
consider removing these packages 
"react-native-tailwindcss": "^1.1.11",
"tailwind-react-native-classnames": "^1.5.1"

**Review by justinmihigo — APPROVED** [review]
Why did you implement features when you are not done with the chore

**Comment by chrysloi** [comment]
Hi @twizerim, you do need to do squash on this PR. you have around 7 commits.

**Review by Emmyfrank — APPROVED** [review]

File: package-lock.json
Comment (Adadie) [inline package-lock.json:1]: Remove this file. It's a duplicate of yarn-lock

File: app/index.tsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+import Appointiment from "./Appointment/screen"
+import Appoin from "./Appointment/appintment"
 
 const App = () => {
   return (
-    <View style={containerStyle}>
-      <Text style={titleStyle}>Medica App</Text>
-      <Button title={"Click me"} />
+    <View className='flex-1 items-center justify-center bg-white'>
+      {/* <Appointiment/> */}
```
Comment (Adadie) [inline app/index.tsx]: Remove comments

File: app/Appointment/screen.tsx
Comment (Adadie) [inline app/Appointment/screen.tsx:1]: We should only have files related to styles in this PR. Remove those that aren't relevant

File: app/Appointment/reschedul.tsx
Comment (Adadie) [inline app/Appointment/reschedul.tsx:1]: Not relevant to styles

File: app/Appointment/appintment.tsx
Comment (Adadie) [inline app/Appointment/appintment.tsx:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/voice.png
Comment (Adadie) [inline app/Appointment/Medica/voice.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/video.png
Comment (Adadie) [inline app/Appointment/Medica/video.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/quinn.png
Comment (Adadie) [inline app/Appointment/Medica/quinn.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/profile.png
Comment (Adadie) [inline app/Appointment/Medica/profile.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/message.png
Comment (Adadie) [inline app/Appointment/Medica/message.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/logo.png
Comment (Adadie) [inline app/Appointment/Medica/logo.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/keegan.png
Comment (Adadie) [inline app/Appointment/Medica/keegan.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/home.png
Comment (Adadie) [inline app/Appointment/Medica/home.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/history.png
Comment (Adadie) [inline app/Appointment/Medica/history.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/dranke.png
Comment (Adadie) [inline app/Appointment/Medica/dranke.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/drake.png
Comment (Adadie) [inline app/Appointment/Medica/drake.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/component/checkbox.tsx
Comment (Adadie) [inline app/Appointment/Medica/component/checkbox.tsx:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/component/cardss.tsx
Comment (Adadie) [inline app/Appointment/Medica/component/cardss.tsx:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/component/cards.tsx
Comment (Adadie) [inline app/Appointment/Medica/component/cards.tsx:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/Raul.png
Comment (Adadie) [inline app/Appointment/Medica/Raul.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/Search.png
Comment (Adadie) [inline app/Appointment/Medica/Search.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/More.png
Comment (Adadie) [inline app/Appointment/Medica/More.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/Frame.png
Comment (Adadie) [inline app/Appointment/Medica/Frame.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/Article.png
Comment (Adadie) [inline app/Appointment/Medica/Article.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/Arrow.png
Comment (Adadie) [inline app/Appointment/Medica/Arrow.png:1]: Not relevant to the task of setting up styles

File: app/Appointment/Medica/Appoint.png
Comment (Adadie) [inline app/Appointment/Medica/Appoint.png:1]: Not relevant to the task of setting up styles

**Review by Adadie — CHANGES_REQUESTED** [review]

**Comment by Adadie** [comment]
Checks are also failing

**Comment by Adadie** [comment]
I believe that atleast on action should pass.  The one for build and test

File: .env
Comment (Adadie) [inline .env:1]: Remove package-lock.json from your .env

**Review by Emmyfrank — APPROVED** [review]

**Review by chrysloi — APPROVED** [review]
Great!

**Review by 911samuel — APPROVED** [review]
nice one, and i see you worked on feedback provided

**Review by fredshema — APPROVED** [review]

**Pull request merged**